easybuild/easyconfigs/p/Pillow/Pillow-3.2.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow'
+version = '3.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pillow.readthedocs.org/'
+description = """Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
+ PIL is the Python Imaging Library by Fredrik Lundh and Contributors."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('libjpeg-turbo', '1.4.2','-NASM-2.12.01'),
+    ('libpng', '1.6.21'),
+    ('zlib', '1.2.8'),
+    ('LibTIFF', '4.0.6'),
+    ('freetype', '2.6.3'),
```

Comment:
@FokkeDijkstra hmm, looks at this in more detail again, there indeed seems to be something fishy going on with the reported error, since several of the mentioned software packages have nothing to do with `scikit-image` or `Pillow`, at first sight... I'll take a closer look and try and figure out what's going on here.

W.r.t. the dependency version hell: you're right, and we've been trying to be more consistent in that department recently. There are also some ideas with regard to a configuration option like `--freeze-deps`, where you can specify which version of (common) dependencies you'd like to stick to, such that EasyBuild can error out if it detects you're trying to install a different version which may make the generated modules incompatible with what's already there for that toolchain. It hasn't materialised yet though, someone will have to actually implement that...